src/awsexplorer/localExplorer.ts

```diff
@@ -36,7 +56,11 @@ export function createLocalExplorerView(rootNodes: RootNode[]): vscode.TreeView<
 
     // Cloud9 will only refresh when refreshing the entire tree
     if (isCloud9()) {
-        rootNodes.forEach(node => node.onDidChangeChildren?.(() => treeDataProvider.refresh(node)))
+        rootNodes.forEach(node => {
+            const refresh = throttle(() => treeDataProvider.refresh(node), 10)
```

Comment:
One-line comment explaining the intent would be helpful. 

    // Delay(?) each node refresh by 10ms because ...